kafka/rdkafka/__init__.py

```diff
@@ -1,6 +1,8 @@
 import logging
+from time import sleep
 
 from kafka import base
+from .config import default_config
```

Comment:
Looks like this line made Travis-CI fail. It was unable to import it for some reason.